src/code/middleware/its-log.js

```diff
@@ -162,7 +163,7 @@ function createLogEntry(loggingMetadata, action, nextState){
   delete context.type;
   delete context.meta;
 
-  context = makeMutable(context);
+  context = makeMutable(cloneDeep(context));
```

Comment:
Oh, I should have probably made this comment a couple of lines below - make mutable just makes the context mutable, and then it changes the values `0` and `1` into `male` and `female` in the following 3 lines. I was seeing those changes then persist through to the state.